README.md

```diff
@@ -31,6 +31,14 @@ This should result in some (stopped) VMs created by tripleo-quickstart on the
 local virthost and some other dependencies installed.
 
 - `./03_ocp_repo_sync.sh`
+
+After this step, you can run the [facet](https://github.com/metalkube/facet)
+server with:
+
+```
+$ go run $GOPATH/src/github.com/metalkube/facet/main.go server
```

Comment:
This is fine for now, but I was thinking we'd probably have a wrapper script which can start the API, UI and any dependencies, that can be a follow-up though.

Reply:
This does start everything. The UI is served at `/`, and the REST API is served at `/api`. The go server is both a static asset server, and a REST API server.

It can be also be started using the binary:

```
$ $GOPATH/src/github.com/metalkube/facet/bin/facet server
```

Reply:
Ack, Ok maybe later we have `05_run_metalkube_installer.sh` or something and we remove the current 05/06 scripts, the README seems fine for now :+1: